docs/queues.txt

```diff
@@ -86,22 +86,15 @@ how to handle failed jobs:
      - **Required** Queue driver to use. The value of
        this property must be ``mongodb``.
 
-   * - ``connection``
+   * - ``database``
      - Database connection used to store jobs. It must be
        a ``mongodb`` connection. The driver uses the default connection
        if a connection is not specified.
 
-   * - ``collection``
+   * - ``table``
      - Name of the MongoDB collection to store failed
        jobs. The value is ``failed_jobs`` by default.
 
```

Comment:
according to the [registerFailedJobServices](https://github.com/laravel/framework/blob/8ffd13c6c66d431450a08d78005feac2c332c72b/src/Illuminate/Queue/QueueServiceProvider.php#L268-L269) method in the `QueueServiceProvider` class, the keys must be `database` and `table` instead of `connections` and `collection`.